elm-package.json

```diff
@@ -4,12 +4,14 @@
     "repository": "https://github.com/datasciencebr/jarbas.git",
     "license": "BSD3",
     "source-directories": [
-        "jarbas/frontend/elm/"
+        "./jarbas/frontend/elm/",
+        "./tests/"
```

Comment:
I agree with your comment in the PR: it is more meaningful to move this directory to `jarbas/frontend/tests`.